tests/test_vlan.py

```diff
@@ -261,8 +260,9 @@ def test_AddMaxVlan(self, dvs, testlog):
         # create max vlan
         vlan = min_vid
         while vlan <= max_vid:
-            self.create_vlan(str(vlan))
+            self.create_vlan(str(vlan), 0)
             vlan += 1
+        time.sleep(600)
```

Comment:
why do we need to sleep for 10 minutes?

Reply:
I skip the sleep on create_vlan and remove_vlan function to shorten the test time. And call time.sleep to wait for the system to become stable after create/remove vlan.

I am waiting the test results on Jenkins. It seems 240 seconds does not enough to finish the test on 1st commit, so I extend the sleep time (10 minutes)  to retest to check if the sleep time is enough or not

Reply:
@lguohan The test is passed, it seems 10 minutes can finish the test. I will try other sleep time to find the most short time for the test.

`test_vlan.py::TestVlan::test_AddMaxVlan PASSED                           [ 83%]`

Reply:
@lguohan, @stcheng the final test time is : 
 - take 6 minutes to add 4k vlan
 - take 7 minutes to remove 4k vlan

The test result of all VLAN cases are PASSED.
test_vlan.py::TestVlan::test_VlanAddRemove PASSED                        [ 75%]
test_vlan.py::TestVlan::test_MultipleVlan PASSED                         [ 76%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectKeyPrefix[test_input0-0] PASSED [ 77%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectKeyPrefix[test_input1-0] PASSED [ 78%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectKeyPrefix[test_input2-0] PASSED [ 79%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectKeyPrefix[test_input3-1] PASSED [ 79%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectValueType[test_input0-0] PASSED [ 80%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectValueType[test_input1-0] PASSED [ 81%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectValueType[test_input2-0] PASSED [ 82%]
test_vlan.py::TestVlan::test_AddVlanWithIncorrectValueType[test_input3-1] PASSED [ 83%]
test_vlan.py::TestVlan::test_AddMaxVlan PASSED                           [ 83%]